src/android/test.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * test module
+ *
+ * Created by Appcelerator, Inc
+ * Copyright (c) 2016 Appcelerator. All rights reserved.
+ */
+import { View } from './view';
+
+export default class Test {
+
+	constructor (app, module) {
+		this.app = app;
+		this.module = module;
+	}
+
+	startup () {
+		this.app.logger.info(this.module.id + ' startup');
+	}
+
+	shutdown () {
+		this.app.logger.info(this.module.id + ' shutdown');
+	}
+
+	static createView (opts = {}) {
```

Comment:
Less code the better. More code means more I/O, more parsing, more evaluation, more memory, more execution calls.